front_end/core/host/RNPerfMetrics.ts

```diff
@@ -70,33 +71,33 @@ class RNPerfMetrics {
   }
 
   registerGlobalErrorReporting(): void {
-    window.addEventListener('error', event => {
-      this.sendEvent({
-        eventName: 'Browser.UnhandledError',
-        params: {
-          type: 'error',
-          message: event.message,
-          error: event.error instanceof Error ? event.error : null,
-        }
-      });
-    }, {passive: true});
-
-    window.addEventListener('unhandledrejection', event => {
```

Comment:
Could it be that `unhandledrejection` event is emitted, but there was no `console.error` call?

Reply:
Wow good catch! You're right. Updated and tested with the following:

```
setTimeout(() => {console.error('printed console error')}, 1000);
setTimeout(() => {throw new Error('thrown Error')}, 2000);

new Promise((r, rej) => {
  rej(new Error('rejected promise as error'));
});

new Promise((r, rej) => {
  rej(('rejected promise as string'));
});
```